apps/src/code-studio/initApp/clientApi.js

```diff
@@ -185,7 +185,9 @@ var base = {
   }
 };
 
-
+function errorString(request, status, error) {
+  return new Error(`httpStatusCode: ${request.status}; status: ${status}; error: ${error}`);
+}
```

Comment:
as a follow-up task I'll start using this new logging format elsewhere in this file, but I wanted to ship this change first before making a broader change which might break other tests.